### PR TITLE
Change in FAQs

### PR DESCRIPTION
**Pull request by gjyotin305**
Change in Academic FAQs and Registration FAQs

### CODE REVIEW

**Comment by gjyotin305** [comment]
@rohitkori

File: cs/academics_faqs.php [hunk 1]
```diff
@@ -298,6 +298,7 @@
                         <li>B.Tech. (Electrical Engineering) Program</li>
                         <li>B.Tech. (Mechanical Engineering) Program</li>
                         <li>B.Tech. (Biotechnology) Program</li>
+                        <li>B.Tech. (Artificial Intelligence and DataScience) Program</li>
```
Comment (rohitkori) [inline cs/academics_faqs.php:301]: Add other branches also

File: cs/academics_faqs.php [hunk 2]
```diff
@@ -328,10 +329,10 @@
                     </ol>
                     Institute also provide non degree program.
                     <br>
-                    Follow this link for further information: <a href="http://iitj.ac.in/academics/index.php?id=programs" target="_blank">http://iitj.ac.in/academics/index.php?id=programs</a>
+                    Follow this link for further information: <a href="https://iitj.ac.in/academics/index.php?id=acad_program&&prog=46" target="_blank">http://iitj.ac.in/academics/index.php?id=programs</a>
```
Comment (rohitkori) [inline cs/academics_faqs.php:335]: Change http to https

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Instructions
 1. Checkout the repository
-* Run `docker-compose up`
+* Run `docker compose up`
```
Comment (rohitkori) [inline README.md]: Revert this change

File: cs/academics_faqs.php (hunk 1 above)
Comment (rohitkori) [reply cs/academics_faqs.php:301]: also change the number of Btech programs
